### PR TITLE
2734 addresses test

### PR DESCRIPTION
**Pull request by peter-freshworks**
*Issue #:* /bcgov/entity#2734

*Description of changes:*
Tests for offices in COLIN incorporation application

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: colin-api/tests/unit/api/test_incorporation_application.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import json
+
+from registry_schemas.example_data import INCORPORATION_FILING_TEMPLATE
+
+from tests import oracle_integration
+
+
+@oracle_integration
+def test_incorporate_bcomp(client):
+    """Assert that a new corp number can be retrieved from COLIN."""
```
Comment (kialj876) [inline colin-api/tests/unit/api/test_incorporation_application.py]: update comment for this method!

File: colin-api/tests/unit/api/test_incorporation_application.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+    offices = rv.json
+
+    assert offices['recordsOffice']
+    assert offices['registeredOffice']
+
+
+@oracle_integration
+def test_incorporate_coop(client):
+    """Assert that a new corp number can be retrieved from COLIN."""
```
Comment (kialj876) [inline colin-api/tests/unit/api/test_incorporation_application.py]: update comment

File: colin-api/tests/unit/api/test_incorporation_application.py (hunk 1 above)
Comment (peter-freshworks) [reply colin-api/tests/unit/api/test_incorporation_application.py]: done

File: colin-api/tests/unit/api/test_incorporation_application.py (hunk 2 above)
Comment (peter-freshworks) [reply colin-api/tests/unit/api/test_incorporation_application.py]: done

**Pull request merged**